src/internal/mediaUtil.ts

```diff
@@ -74,3 +74,27 @@ export function validateViewImagesInput(uriList: media.ImageUri[]): boolean {
   }
   return true;
 }
+
+/**
+ * Returns true if the scan barcode param is valid and false otherwise
+ */
+export function validateScanBarCodeInput(barCodeConfig: media.BarCodeConfig): boolean {
+  if (barCodeConfig) {
+    if (
+      barCodeConfig.barCodeType !== null &&
+      barCodeConfig.barCodeType !== undefined &&
+      (barCodeConfig.barCodeType < media.BarCodeType.All ||
+        barCodeConfig.barCodeType > media.BarCodeType.TwoDimensional)
+    ) {
+      return false;
```

Comment:
Shouldn't we also log an error to the console or in some other way communicate to the developer exactly why we are rejecting their call?

Reply:
We will be returning INVALID_ARGUMENTS error code in the error response in case the user has passed invalid values in the barcodeConfig param.